.travis.yml

```diff
@@ -33,5 +32,5 @@ deploy:
   on:
     tags: true
     repo: CERNDocumentServer/cds-dojson
-    python: '2.7'
+    python: '3.6'
```

Comment:
we just have to make sure that this will not affect Videos somehow... if Videos depend on a release, than it is fine!

Reply:
its books branch for now, but eventually we could depend on the version